.github/workflows/prerelease.yml

```diff
@@ -65,9 +65,9 @@ jobs:
           retention-days: 1
 
       - name: Publish Pre-Release
-        uses: softprops/action-gh-release@v1
+        uses: softprops/action-gh-release@v2
         with:
-          tag_name: "latest"
+          name: "Pre-Release"
```

Comment:
why did you removed tag_name?

Reply:
because `latest` is some old tag that nobody is updating. If I understood the action documentation properly, the variable should be default initialized to the branch to which was pushed (which should always be `master` for this action)